pom.xml

```diff
@@ -210,7 +215,7 @@
         <carbon.identity.framework.version>5.11.148</carbon.identity.framework.version>
         <carbon.identity.framework.imp.pkg.version.range>[5.0.0, 6.0.0)
         </carbon.identity.framework.imp.pkg.version.range>
-        <carbon.kernel.version>4.6.1</carbon.kernel.version>
+        <carbon.kernel.version>4.9.0-beta</carbon.kernel.version>
```

Comment:
We can use 4.9.0 here since it's released now.

Reply:
fixed